lib/rpmsg/rpmsg_virtio.c

```diff
@@ -630,6 +642,12 @@ int rpmsg_init_vdev(struct rpmsg_virtio_device *rvdev,
 	rdev->ops.send_offchannel_nocopy = rpmsg_virtio_send_offchannel_nocopy;
 	role = rpmsg_virtio_get_role(rvdev);
 
+	if (config) {
+		rvdev->config = *config;
+	} else {
+		rvdev->config = RPMSG_VIRTIO_DEFAULT_CONFIG;
```

Comment:
What about  just past RPMSG_VIRTIO_DEFAULT_CONFIG as argument in rpmsg_init_vdev function and suppress this condition?

Reply:
We would need some code to handle `config == NULL`. What do you suggest? Return `RPMSG_ERR_PARAM`?